sample/configs/prebid-config-with-module.yaml

```diff
@@ -12,6 +12,9 @@ adapters:
     enabled: true
 metrics:
   prefix: prebid
+analytics:
+  greenbids:
+    enabled: true
```

Comment:
greenbids config should not be merged (or at least make it disabled)

Reply:
removed config for RTD module and analytics adapter from `sample/configs/prebid-config-with-module.yaml`